SetPreferredSinkId/explainer.md

```diff
@@ -82,9 +80,9 @@ Exceptions:
     Returned if switching the audio output device to the new audio device failed.
 ```
 
-This API is accessible from the top-level frame and the same-origin sub frames and allows modification of the default audio output for both
-the top-level frame and all sub frames, regardless of their origins. However, it’s important to note that this change does not affect custom
-audio outputs specified using the setSinkId method in media element or audio context.
+This API is accessible from the top-level frame and allows modification of the default audio output for both the top-level frame and all sub frames, 
```

Comment:
Should we also mention what happens when setPreferredSinkId is called from a sub frame?  I assume it throws an exception?

Reply:
Updated the above error message 'Returned if a sub frame tries to call the API.'.